make keyboard shortcuts consistent

johan456789 opened the pull request:
- capitalize first letter of each key and remove spaces between keys and plus signs (consistent with #416)
- use "Key-click" (common in the manual) instead of "Key+click" (less common)

Danika-Dakika commented:
Looks good!

Danika-Dakika commented:
I think I gathered that much. 😉 My concern was more along the lines of -- "Is what is happening here a good decision? Are we just having bots do our editing now? Are these commit notes correct?"

johan456789 committed:
fix: add kbd tag and make format consistent

Co-authored-by: google-labs-jules[bot] <161369871+google-labs-jules[bot]@users.noreply.github.com>

johan456789 commented:
Yes, jules is an AI bot made by google. I'm experimenting with those tools and I forgot to squash those commits. They do have a tendency to hallucinate in their commit notes.

I've squashed the commits and removed the confusing notes. I'd remember to do that from now on.

>  "Is what is happening here a good decision? Are we just having bots do our editing now? Are these commit notes correct?"

My opinion is as long as the final submission is good work, I don't see a problem here. It's quite useful to let the AI do the tedious work.

johan456789 committed:
Update display-issues.md

johan456789 commented on src/platform/mac/display-issues.md:
```diff
@@ -5,12 +5,14 @@
 ### Changing the Driver From Terminal.app
+
 Older Anki versions did not provide an option in the preferences, but allowed
 you to adjust the driver by opening Terminal.app, then pasting the following and hit <kbd>Enter</kbd>:
```
> Maybe we can agree on certain styling on the code side or agree to certain linter rules. Maybe it's worth to open a new issue regarding integrating the linter?

Yes, opening an issue for this would be great.

johan456789 committed:
Merge branch 'main' into style-kbd-tags

Danika-Dakika commented:
Thank you!
